cmake/modules/cutelystqt5-config.cmake.in

```diff
@@ -10,4 +10,4 @@ SET(CutelystQt5_LIBRARY "@CMAKE_INSTALL_LIBDIR@/libcutelyst-qt5.so" CACHE FILEPA
 SET(CutelystQt5_INCLUDE_DIR "@CMAKE_INSTALL_PREFIX@/include/cutelyst-qt5/" CACHE PATH "Include path for Cutelyst")
 SET(CutelystQt5_FOUND "TRUE")
 
-include("${CMAKE_CURRENT_LIST_DIR}/CutelystQt5Targets.cmake")
+include("${prefix}/cmake/CutelystQt5Targets.cmake")
```

Comment:
I'd thought that it better be separated into `{prefix}/cmake` directory. {prefix} is actually the CMAKE_INSTALL_PREFIX.